sql/src/test/java/io/crate/operation/collect/LuceneDocCollectorTest.java

```diff
@@ -177,7 +177,16 @@ public void testLimitWithoutOrder() throws Exception{
     }
 
     @Test
-    public void testOrderedWithLimit() throws Exception{
+    public void testKillCollect() throws Exception {
+        collectingProjector.rows.clear();
+        LuceneDocCollector docCollector = createDocCollector(null, 10, orderBy.orderBySymbols());
+        docCollector.kill();
+        docCollector.doCollect();
```

Comment:
this should not be allowed. put an assertion into docollect that it is not killed already